stage1/units/units/halt.target

```diff
@@ -1,4 +0,0 @@
-[Unit]
-Description=Halt
-DefaultDependencies=no
-AllowIsolate=yes
```

Comment:
This unit is still referenced in many places, but it is removed here. How does it work?